frontend/src/graphql/queries.js

```diff
@@ -127,3 +127,20 @@ export const communities = gql`
     }
   }
 `
+
+export const queryTransactionLink = gql`
+  query($code: String!) {
+    queryTransactionLink(code: $code) {
+      amount
+      memo
+      createdAt
+      validUntil
+      user {
+        email
+        firstName
+        lastName
+        publisherId
```

Comment:
we need to see if the `publisherId` is the id of the guy who recommended this user. you need to use `user.id` and we need to use a new field instead of `publisherId`, since that is elopage and different from our ids.

Reply:
Changed it to the user.id